Source/WebKit2/Shared/CoordinatedGraphics/CoordinatedGraphicsScene.cpp

```diff
@@ -67,7 +67,7 @@ CoordinatedGraphicsScene::CoordinatedGraphicsScene(CoordinatedGraphicsSceneClien
     : m_client(client)
     , m_isActive(false)
     , m_rootLayerID(InvalidCoordinatedLayerID)
-#if PLATFORM(INTEL_CE)
+#if PLATFORM(INTEL_CE) || PLATFORM(WPE)
```

Comment:
By adding PLATFORM(WPE) you're basically changing the behaviour for all the platforms WPE is running on. Actually it would be the same than removing the #if and just setting the value to transparent.